src/views/preview/project-view.jsx

```diff
@@ -26,8 +26,7 @@ const CanceledDeletionModal = require('../../components/login/canceled-deletion-
 const NotAvailable = require('../../components/not-available/not-available.jsx');
 const Meta = require('./meta.jsx');
 const {
-    onProjectShared,
-    onProjectLoaded
+    onProjectShared
```

Comment:
I removed this one because of the linter. I was thinking to have a separate PR that removes calls to user-guiding as a whole so that things don't get mixed up. Can do it here if you want

Reply:
It can be done in another PR against the integration branch